packages/widgets/src/blocks/legacy-widget/editor.scss

```diff
@@ -57,7 +57,7 @@
 			font-family: system-ui;
 			background-color: transparent;
 			box-sizing: border-box;
-			border: 1px solid $gray-700;
+			border: $border-width solid $gray-700;
 			border-radius: 3px;
```

Comment:
Also unrelated, but I cannot help myself. Should this be `$radius-small + $border-width`?